packages/block-editor/src/components/tabbed-sidebar/style.scss

```diff
@@ -4,10 +4,6 @@
 	flex-direction: column;
 	flex-grow: 1;
 	overflow: hidden;
-
-	@include break-medium() {
-		width: 350px;
-	}
```

Comment:
This shouldn't have been fixed at 350, as it shouldn't care what the parent width is. This would need to be removed to use it within the right side setting sidebar anyways.